app_radio_module/src/transmitter.c

```diff
@@ -81,6 +90,36 @@ static void lora_tx_task(void *, void *, void *) {
     }
 }
 
+typedef struct {
+    double latitude;
+    double longitude;
+    double altitude;
+} gnss_data_simple;
```

Comment:
Could've sworn I made this a struct in our types.h file. Anyways, put this declaration in there. Could name it gnss_pos instead of gnss_data_simple. Make sure it follows launch_core library standards though so it would end up being l_gnss_data_simple_t. Also make it a packed struct.

Reply:
I would also recommend making altitude a float. Dont need as much precision for altitude, and generally its very difficult to get an accurate altitude measurement. Its difficult to get good accuracy when measuring on the vertical plane because of geometry reasons

Reply:
resolved in https://github.com/RIT-Launch-Initiative/FSW/pull/85/commits/191841cd72fff3a4adf71837e8d04d54f3933c0d and
https://github.com/RIT-Launch-Initiative/FSW/pull/85/commits/3ef4d6f77bfb336a302dc3083e3cc17d0c5e810c